modiscolite/meme_writer.py

```diff
@@ -0,0 +1,158 @@
+# Create a class that can be used to write MEME files.
+# Authors: Ivy Raine <ivy.ember.raine@gmail.com>, Jacob Schreiber <jmschreiber91@gmail.com>
+from typing import List, Optional
+import numpy as np
+
+from os import PathLike
+
+class MEMEWriterMotif:
+	"""Class for handling Motif for MEME file writing."""
+
+	def __init__(
+		self,
+		name: str,
+		probability_matrix: np.ndarray,
+		# `alphabet_length` is known as 'alength' in MEME Suite.
```

Comment:
I'd keep comment lines out of the __init__ itself and instead have a Parameters section in the class docstring.

Reply:
Done